src/search/node/FindInFilesDomain.js

```diff
@@ -187,8 +188,26 @@ maxerr: 50, node: true */
         projectCache = [];
     }
 
+
     /**
-     * Get the contents of a file given the path
+     * Gets the file size in bytes.
+     * @param   {string} fileName The name of the file to get the size
+     * @returns {Number} the file size in bytes
+     */
+    function getFilesizeInBytes(fileName) {
+        var stats = {};
```

Comment:
you don't really need to init with an empty object here, it'll get overridden in try-catch anyway